final_project/snapvisite/tests/tests_account/test_account.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from django.contrib.auth import get_user_model
+profile = get_user_model()
+
+
+def test_new_user(new_user):
+    assert new_user.user_name == 'nickname'
```

Comment:
to jest testowanie czy factorka działa, nie wnosi to nic do projektu, chyba, że chciałeś sobie potestować factorki ot tak